README.md

```diff
@@ -33,6 +33,31 @@ un-block it. The benchmarks mentioned above that state `Used` for Reconciliation
 These two bencharmks are written in a way that doesn't allow for reconciliation to be implemented. To take
 advantage of the reconciliation loop, these two benchmarks need to be rewritten.
 
+## Optional workload images
+Optional locations for workload images can now be added easily without the need to rebuild the operator.
+To do so in the workload args section of the CR add image: [location]
+NOTE: This is not a required arguement. If omitted it will default to the currently verified workload image.
```

Comment:
This doesn't work for ycsb, maybe we should add that to note as well?

Reply:
Noted in the Readme